applications/app/controllers/YoutubeController.scala

```diff
@@ -0,0 +1,33 @@
+package controllers
+
+import com.gu.contentapi.client.model.v1.ItemResponse
+import common._
+import contentapi.ContentApiClient
+import model.{CacheTime, Cached}
+import play.api.libs.ws.WSClient
+import play.api.mvc.{Action, AnyContent, BaseController, ControllerComponents}
+
+import scala.concurrent.Future
+import scala.util.{Failure, Success}
+
+class YoutubeController(contentApiClient: ContentApiClient, wsClient: WSClient, val controllerComponents: ControllerComponents) extends BaseController with Logging with ImplicitControllerExecutionContext {
+
+  def getAtomId(id: String): Action[AnyContent] = Action.async { implicit request =>
```

Comment:
Might be worth naming the id parameter here `youtubeID` instead to make this a bit clearer?